### DISCUSSION TITLE
How to disable certbot permanently?

### DISCUSSION THREAD

**Discussion by egze** [caprover/caprover#2196] [Q&A]
I’ve set up CapRover behind a Cloudflare tunnel with SSL fully managed by Cloudflare.  

I noticed that a container running `caprover/certbot-sleeping:v2.11.0` is active. Is there a way to disable it permanently?

**Comment by githubsaturn** [comment]
You can't disable the container. It's just a `sleep` process. It does not consume any ram / CPU if you don't use it.
